### PR TITLE
red-tldr 0.4 (new formula)

### PR DESCRIPTION
**Pull request by Rvn0xsy**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/red-tldr.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    system "red-tldr"
```
Comment (carlocab) [inline Formula/red-tldr.rb:15]: Can we check some output here? Can we also test some functionality, e.g. `red-tldr search`?

Comment (Rvn0xsy) [reply Formula/red-tldr.rb]: Thanks for your advice

**Review by Rvn0xsy — COMMENTED** [review]
I have added new test cases in the code

File: Formula/red-tldr.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    system "red-tldr", "search", "mimikatz"
```
Comment (carlocab) [inline Formula/red-tldr.rb:15]: Can we check some output here?

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @Rvn0xsy for your contribution!

**Comment by Rvn0xsy** [comment]
> Thanks @Rvn0xsy for your contribution!

It's my pleasure :)

**Pull request closed without merging**